ansible/roles/keycloak-deploy/defaults/main.yml

```diff
@@ -77,6 +77,8 @@ keycloak_custom_ds: '
         </validation>
     </datasource>'
 
+keycloak_heap_size: Xmx512m
```

Comment:
Since we don't want users making typo - Can we make this 512m and then add Xmx in the template?

Reply:
Done